frontend/src/metabase/static-viz/lib/numbers.ts

```diff
@@ -62,10 +62,13 @@ export const formatNumber = (number: number, options?: NumberFormatOptions) => {
 
   const format = createFormat(compact);
 
+  const separatorMap = {
```

Comment:
Make it similar to the app viz logic
https://github.com/metabase/metabase/blob/4d0cdf419f10faf210170c9ad59c6b008e3426c9/frontend/src/metabase/lib/formatting/numbers.tsx#L213-L225